admin-portal/src/types/MedicalNeeds.ts

```diff
@@ -0,0 +1,29 @@
+import { Quotation } from "./Quotation";
+
+export type MedicalNeed = {
```

Comment:
Let's use interfaces to keep consistent as we have used interfaces for other types.

Reply:
Also, let's make one file per interface.